src/python/grpcio/grpc/aio/_channel.py

```diff
@@ -478,11 +478,13 @@ async def channel_ready(self) -> None:
             await self.wait_for_state_change(state)
             state = self.get_state(try_to_connect=True)
 
+    # pylint: disable=arguments-differ,unused-argument
     def unary_unary(
         self,
         method: str,
         request_serializer: Optional[SerializingFunction] = None,
         response_deserializer: Optional[DeserializingFunction] = None,
+        registered_method: Optional[bool] = False,
```

Comment:
Please add a TODO noting that we should support `registered_method` under asyncio and a comment explaining why this is currently a no-op.